recipe/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "sis-extras" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sis_extras-{{ version }}.tar.gz
+  sha256: e7e1892ab5eef36e38880dc9b636597dce83162df4d51f3f7f509a7e792e8a73
+
+build:
+  skip: True  # [py>38]
+  # no streamlit on s390x
+  skip: True  # [s390x]
```

Comment:
Can reduce this 
```suggestion
  # no streamlit on s390x
  skip: True  # [py>38 or s390x]
```

Reply:
I wanted to distinguish them by the associated comments.
(I'm not so bothered.)